outlook/src/taskpane/components/Company/CompanySection/CompanySection.tsx

```diff
@@ -49,48 +49,40 @@ class CompanySection extends React.Component<CompanySectionProps, CompanySection
             isCollapsed: false,
             isLoading: false
         };
-
-        this.enriched = false;
-
         this.companyCache = new CompanyCache(2, 200, 25);
     }
 
     private companyCache: CompanyCache;
-    private enriched: boolean;
 
-    private enrichCompanyForPartnerRequest = () => {
+    private enrichAndCreate = () => {
 
         if (!this.props.partner.email)
         {
             let enrichmentInfo = new EnrichmentInfo(EnrichmentInfoType.EnrichContactWithNoEmail,
-                _t("This contact has no email address, no company could be enriched."));
+                _t("This company has no email address and couldn't be enriched."));
```

Comment:
Is it translated in 14.3 ?

Reply:
Same thing with translations, must be included in the forward port.